components/kyma-environment-broker/internal/broker/instance_create.go

```diff
@@ -251,6 +251,9 @@ func (b *ProvisionEndpoint) extractERSContext(details domain.ProvisionDetails) (
 	if ersContext.SubAccountID == "" {
 		return ersContext, errors.New("subAccountID parameter cannot be empty")
 	}
+	if ersContext.UserID == "" {
```

Comment:
Why the User is mandatory?
if we really need this check (and userID is mandatory) then we need to adjust documentation about provisioning SKR and also integration tests:
 - SKR fast integration test (codebase in kyma repository)
 - e2e spinnaker pipelines in the tests/e2e folder

Reply:
Regarding the kyma upgrade process - we need to check how many instances we have without userID and how we can proceed with them.

Reply:
Added tests for: FIT and spinnaker. 
Added doc